src/taipy/config/_json_serializer.py

```diff
@@ -36,8 +41,27 @@ def _read(cls, filename: str) -> _Config:
 
     @classmethod
     def _serialize(cls, configuration: _Config) -> str:
-        return json.dumps(cls._str(configuration), ensure_ascii=False, indent=0, check_circular=False)
+        return json.dumps(
+            cls._str(configuration), ensure_ascii=False, indent=0, check_circular=False, cls=_CustomEncoder
+        )
 
     @classmethod
     def _deserialize(cls, config_as_string: str) -> _Config:
         return cls._from_dict(cls._pythonify(dict(json.loads(config_as_string))))
+
+
+class Json:
+    pass
+
+
+class _CustomEncoder(json.JSONEncoder):
+    def default(self, o: Any) -> Json:
+        if isinstance(o, Enum):
+            result = o.value
+        elif isinstance(o, datetime):
+            result = f"{o.isoformat()}:datetime"
+        elif isinstance(o, timedelta):
+            result = f"{_timedelta_to_str(o)}:timedelta"
+        else:
+            result = json.JSONEncoder.default(self, o)
+        return result
```

Comment:
I agree with Giang. 
Methods _stringify and _pythonify play the roles of custom encoder and decoder. I believe _CustomEncoder and Json classes are not really necessary.